docs/_data/navigation.yml

```diff
@@ -50,4 +50,9 @@
     - title: "Forks"
       url: "/housekeeping-forks"
 - title: "Recommendations"
-  url: "/recommendations"
+  url: "/recommendations-tokenless-auth"
+  subnavigation:
+    - title: "Tokenless Authentication"
+      url: "/recommendations-tokenless-auth"
+    - title: "Legacy Teams"
+      url: "/recommendations-teams-legacy"
```

Comment:
I’d have named the URL `recommendations-legacy-teams`. I think that we wanted to have the navigation title (here, *Recommendations*) as the first component of the URL and have the rest match the subnavigation title (here, *Legacy Teams*).

Maybe, we should consider moving all the pages belonging to a navigation item into a subdirectory. In this way, the URLs would automatically be in the form `/recommendations/legacy-teams`, and this might make naming the URLs more consistent.